test/index.test.js

```diff
@@ -202,15 +200,14 @@ test('scan objects, concurrency=1', function(assert) {
         var keys = Object.keys(fixtures);
         keys = keys.filter(function(k) { return k[0] === '0'; });
         assert.equal(found.length, keys.length, 'retrieved all objects');
-        assert.deepEqual(found, keys.sort(), 'found all expected keys in ascending order');
+        assert.deepEqual(found.sort(), keys.sort(), 'found all expected keys');
```

Comment:
Do we have anything depending on the ascending order? If so, we should fix the code and not change the tests.

Reply:
[this](https://stackoverflow.com/questions/4102115/does-the-listobjects-command-guarantee-the-results-are-sorted-by-key)

Reply:
Is there a way for us to preserve the test case as it was earlier?